src/DotnetDocument/Configuration/DocumentationOptions.cs

```diff
@@ -353,6 +353,11 @@ public ExtendedSummaryDocumentationOptions()
         /// Gets or inits the value of the include inheritance
         /// </summary>
         public bool IncludeInheritance { get; init; } = true;
+
+        /// <summary>
+        /// Gets or inits the value of include type parameters
+        /// </summary>
+        public bool IncludeTypeParams { get; init; } = true;
```

Comment:
We could do something similar to what has been done for the `MethodDocumentationOptions` class [here](https://github.com/p1va/dotnet-document/blob/37b42cfc2c8ef984b17933a765cd77c663c8fad8/src/DotnetDocument/Configuration/DocumentationOptions.cs#L131) and add the `ParamsDocumentationOptions` options directly on the `ClassDocumentationOptions` like
```csharp
public class ClassDocumentationOptions : MemberDocumentationOptionsBase
{
    ...
    
    /// <summary>
    /// Gets or sets the value of the type parameters
    /// </summary>
    public ParamsDocumentationOptions TypeParameters { get; set; } = new();
    
    ...
}
```